### ISSUE TITLE
(URGENT) Crash in Relaton IEEE

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-ieee#7]
```adoc
* [[[IEEE.754.2008,IEEE 754:2008]]], _IEEE Standard 754, Standard for Binary Floating-Point Arithmetic_
```

```
/Users/me/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/relaton-ieee-1.7.4/lib/relaton_ieee/hit_collection.rb:27:in `block in initialize': end pattern with unma (if / (RegexpError)^NoPatent OwnerContact for LicensePatent 
```

**Comment by ronaldtse** [comment]
Metanorma should never crash on a failed Relaton fetch (ping @opoudjis )

**Comment by andrew2net** [comment]
@ronaldtse I'm not able to reproduce it. Which Ruby version and OS do you use?

**Comment by ronaldtse** [comment]
I am on Ruby 2.6.6, macOS.

**Comment by ronaldtse** [comment]
@andrew2net in that PR there is the identical error:

https://github.com/CalConnect/csd-vobject-vformat/pull/2/checks?check_run_id=2373197720

```
[relaton-ieee] ("IEEE 754") fetching...
45
/usr/local/bundle/gems/relaton-ieee-1.7.4/lib/relaton_ieee/hit_collection.rb:27:in `block in initialize': end pattern with unmatched parenthesis: /^NoPatent OwnerContact for LicensePatent 
46
 Serial No. 
47
```

Please help fix this ASAP, thanks.

**Comment by ronaldtse** [comment]
And the page is this:
https://standards.ieee.org/standard/754-2008.html

**Comment by andrew2net** [comment]
Fixed in the version 1.7.5

**Comment by ronaldtse** [comment]
Thanks @andrew2net !
